includes/discovery/mempools/fortimail.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+if ($device['os'] == 'fortimail') {
```

Comment:
Device OS will always be fortimail here, so you should remove this condition.

Reply:
I followed the documentation here: https://docs.librenms.org/Developing/os/Mem-CPU-Information. But if this condition really is obsolete, I'll remove it from my code.

Reply:
If I got it right, the includes/discovery/mempools.inc.php will run all available mempool-definitions against a device. The condition inside a definition takes care that only the relevant SNMP queries are executed. --> Maybe this kind of logic could be done within mempools.inc.php?